src/app/components/team/TeamTags.js

```diff
@@ -466,48 +472,55 @@ class TeamTagsComponent extends Component {
             label={filterLabel}
             tooltip={<FormattedMessage id="teamTags.tooltip" defaultMessage="Filter and sort list" />}
           >
-            <div style={{ marginTop: units(2) }}>
+            <Box mt={2}>
               <SortSelector
                 value={this.state.sort}
                 onChange={this.handleChange.bind(this)}
                 fullWidth
               />
-            </div>
+            </Box>
           </FilterPopup>
         </AlignOpposite>
-        <Card style={{ marginTop: units(2) }}>
-          <CardContent style={{ padding: 0 }}>
-            { tag_texts.length === 0 ?
-              <p style={{ paddingBottom: units(5), textAlign: 'center' }}>
-                <FormattedMessage
-                  id="teamTags.noTags"
-                  defaultMessage="No tags"
-                />
-              </p>
-              : null }
-            {this.tagsList(tag_texts)}
-            <Can permissions={this.props.team.permissions} permission="create TagText">
-              <div style={{ padding: units(2) }}>
-                <TextField
-                  id="tag__new"
-                  onKeyUp={this.handleKeyUp.bind(this)}
-                  onKeyPress={this.handleKeyPress.bind(this)}
-                  label={<FormattedMessage id="teamTags.new" defaultMessage="New tag" />}
-                  style={{ width: '50%' }}
-                />
-                <p>
-                  <Button
-                    onClick={this.handleAddTag.bind(this)}
-                    disabled={this.state.newTag.length === 0}
-                    color="primary"
-                  >
-                    <FormattedMessage id="teamTags.addTag" defaultMessage="Add tag" />
-                  </Button>
-                </p>
-              </div>
-            </Can>
-          </CardContent>
-        </Card>
+        <Box mt={2}>
+          <Card>
+            <Box p={0}>
+              <CardContent>
+                { tag_texts.length === 0 ?
+                  <Box pb={5} display='flex' alignItems='center'>
```

Comment:
Please use single quotes for string props/attributes.

`489:39  error  Unexpected usage of singlequote    jsx-quotes`